playbooks/files/loki/grafana_provisioning.yml.j2

```
@@ -0,0 +1,10 @@
+apiVersion: 1
```

to control the data sources completely from config files, you should add a `prune` option, so grafana removes in its database any DS removed from the configuration
```suggestion
apiVersion: 1

prune: true
```